### PR TITLE
Including clients options into the storybook

### PR DESCRIPTION
**Pull request by LinconOssa**
The stories include the ability to select among Corbanca and Fondecom clients. Present and Cooservunal this has the functionality to exchange between the tokens of each clients.

### CODE REVIEW

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+      Story: StoryFn,
+      context: { args: { formType: keyof typeof tokensWithReference } }
+    ) => (
       <BrowserRouter>
         <TokenContext.Provider
           value={{
-            tokenWithRef: presente,
+            tokenWithRef: tokensWithReference[context.args.formType],
             loading: false,
             handleSubmit: () => {},
```
Comment (Andresbl123) [inline src/pages/people/outlets/color/palette/stories/Palette.stories.tsx:25]: para no dejar esta función vacía podemos utilizar los actinos de Storybook ejem: se importa action
`import { action } from "@storybook/addon-actions";` y aqui hacemos algo como esto: ` handleSubmit: action("handleSubmit"),` cuando se realice el evento, en storybook no lo documenta así : 
image

File: src/pages/people/outlets/color/surfaces/stories/Surfaces.stories.tsx [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+      Story: StoryFn,
+      context: { args: { formType: keyof typeof tokensWithReference } }
+    ) => (
       <BrowserRouter>
         <TokenContext.Provider
           value={{
-            tokenWithRef: presente,
+            tokenWithRef: tokensWithReference[context.args.formType],
             loading: false,
             handleSubmit: () => {},
```
Comment (Andresbl123) [inline src/pages/people/outlets/color/surfaces/stories/Surfaces.stories.tsx:25]: Aquí podemos hacer lo mismo en la recomendación dada

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/pages/people/outlets/color/palette/stories/Palette.stories.tsx]: Resolved
image

File: src/pages/people/outlets/color/surfaces/stories/Surfaces.stories.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/pages/people/outlets/color/surfaces/stories/Surfaces.stories.tsx]: Resolved

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
-    (Story: StoryFn) => (
+    (
+      Story: StoryFn,
+      context: { args: { formType: keyof typeof tokensWithReference } }
+    ) => (
       <BrowserRouter>
         <TokenContext.Provider
           value={{
-            tokenWithRef: presente,
+            tokenWithRef: tokensWithReference[context.args.formType],
```
Comment (carin91) [inline src/pages/people/outlets/color/palette/stories/Palette.stories.tsx:24]: Change the "formType" name for "clientName" or a more appropriate name.

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx [hunk 4]
```diff
@@ -29,6 +35,10 @@ const story = {
 
 const Default = () => <Palette />;
 
+Default.args = {
+  formType: "presente",
```
Comment (carin91) [inline src/pages/people/outlets/color/palette/stories/Palette.stories.tsx:39]: Change the "formType" name for "clientName" or a more appropriate name.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx [hunk 5]
```diff
@@ -1,23 +1,29 @@
...
+import { Palette } from "..";
 import { TokenContext } from "@context/TokenContext";
-import { presente } from "@src/mocks/design/tokensWithReference/presente";
+import { tokensWithReference } from "@src/mocks/design/tokensWithReference";
+import { props } from "./props";
+import { action } from "@storybook/addon-actions";
 
 const story = {
-  components: [Palette],
+  components: Palette,
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/people/outlets/color/palette/stories/Palette.stories.tsx]: Resolved

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx [hunk 6]
```diff
@@ -1,23 +1,29 @@
...
   title: "layouts/people/outlets/color/palette",
   parameters: {
     layout: "fullscreen",
   },
+  argTypes: props,
   decorators: [
-    (Story: StoryFn) => (
+    (
+      Story: StoryFn,
+      context: { args: { formType: keyof typeof tokensWithReference } }
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/people/outlets/color/palette/stories/Palette.stories.tsx]: Resolved
image

File: src/pages/people/outlets/color/palette/stories/Palette.stories.tsx (hunk 3 above)
Comment (LinconOssa) [reply src/pages/people/outlets/color/palette/stories/Palette.stories.tsx]: Resolved
image

File: src/pages/people/outlets/color/palette/stories/props.ts [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { tokensWithReference } from "@src/mocks/design/tokensWithReference";
+
+const props = {
+  formType: {
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/people/outlets/color/palette/stories/props.ts]: Resolved
image

**Review by carin91 — APPROVED** [review]

**Pull request merged**